Sources/EnhancedMediaPlayer/MediaPlayerViewController.swift

```diff
@@ -163,6 +171,18 @@ public class MediaPlayerViewController: UIHostingController<MediaPlayerView> {
         }
     }
 
+    private func addTimeObserver() {
+        timeObserver = self.viewModel.player.addPeriodicTimeObserver(
+            forInterval: Constants.videoUpdateInterval,
```

Comment:
The progress stutters with a 1s update interval. I tried lowering it to 0.1s and it got much better.

Maybe try playing with some values to balance visual update and performance